### PR TITLE
Change cursor to a pointer when hover over read notification

### PR DESCRIPTION
**Pull request by jhadvig**
@jwforres PTAL

Closes https://github.com/openshift/origin-web-console/issues/2293

### CODE REVIEW

**Comment by spadgett** [comment]
I think the fix is to use the default (arrow) cursor when read. Is that what this does? It looks like it makes it a pointer (hand) cursor always.

**Comment by benjaminapetersen** [comment]
So on Master I see what @spadgett mentioned, which is to use the default arrow once it has been read:



This PR will always use the pointer:



@jwforres can you clarify what you mentioned on [this comment](https://github.com/openshift/origin-web-console/issues/2293#issuecomment-338790856)?  Do we just want to eliminate the hover effect entirely?

File: app/styles/_notifications.less [hunk 1]
```diff
@@ -39,9 +39,9 @@ notification-drawer-wrapper {
   }
   // the whole block is clickable, need to set pointer on all of these
   // for the correct visual
```
Comment (benjaminapetersen) [inline app/styles/_notifications.less]: I'm not sure this is quite right, might be that we just want to eliminate the hover (bg blue) as well as return the cursor.

**Comment by spadgett** [comment]
If we switch to the default cursor, I think we'd want to remove the blue hover style as well? But I might have misunderstood @jwforres' comment.

I personally don't have a strong preference here.

**Comment by jwforres** [comment]
weird, i thought i was on master when i checked it...

i would at least see what it looks like without the hover color

**Comment by jhadvig** [comment]
So from [this comment](https://github.com/openshift/origin-web-console/issues/2293#issuecomment-338790856) I was in a impression we want to have to pointer cursor on unread and also read notifications.

**Comment by jhadvig** [comment]
Thinking about it now Im not really sure it would make sense to have the pointer on read notification. If we align on this it would be better to merge @benjaminapetersen [PR](https://github.com/openshift/origin-web-console/pull/2383/files)
If align on the pointer for both cases the I would rather keep the changed background on the hover, just to to have a similar behavior of  read and unread notifications.

@jwforres @spadgett thoughts ?

**Comment by jhadvig** [comment]
Closing this PR in favor of https://github.com/openshift/origin-web-console/pull/2383
@benjaminapetersen fyi :)

**Comment by benjaminapetersen** [comment]
Cool, think mine is gonna get closed as well. Yay discussions!

**Pull request closed without merging**